rails-kickoff-template.rb

```diff
@@ -150,6 +157,39 @@ def setup_sidekiq
   end
 end
 
+def setup_linters
+  gem 'rubocop', require: false
+  gem 'rubocop-rspec', require: false
+  gem 'rubocop-thread_safety', require: false
+  gem 'overcommit'
+
+  after_bundle do
+    #get '/Users/travis/coding/tanooki-rails-templates/.rubocop.yml'
+    copy_file '~/coding/tanooki-rails-template/.rubocop.yml', '.rubocop.yml'
```

Comment:
this will be updated to download from the github repository. Chose it over inheriting from the remote url, because rubocop can change a bit too much from version to version.